io.github.georgewoodall82.jsymphonic.yml

```diff
@@ -0,0 +1,33 @@
+id: io.github.georgewoodall82.jsymphonic
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=host
```

Comment:
again why does it need host?

Reply:
just updated it to remove the host permission and add home, /run/media, /media, and /mnt instead for removable drives. /home is needed because that is where the program gets music from.

Reply:
> /run/media, /media, and /mnt i

home + this is effectively `host`. Why do you need it?

If it is necessary for valid reasons, then just use `host`.

Reply:
Sure just changed it back to host